__tests__/app.test.js

```diff
@@ -77,3 +77,33 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  test("should respond with 200 and a review object with correct review id", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(body).toEqual({ review });
```

Comment:
body is always going to equal review as you just declared it in the line before, it's the same as saying a = b, does b equal a? You need to actually look into the body and make sure the outline of the object looks how you expect, use one of the jest matchers .toMatchObject or .objectContaining to make sure all the keys and values you expect are actually there